docs/reference/settings.rst

```diff
@@ -315,6 +315,17 @@ Specifies the number of items per page shown when viewing an image's usage (see
 
 Specifies the number of images shown per page in the image chooser modal.
 
+``WAGTAILIMAGES_RENDITION_STORAGE``
+-----------------------------------
+
+If specified, this setting allows renditions to be stored using an alternate storage backend (default is None). This value should be a subclass of storages.backends. For example:
+
+.. code-block:: python
+
+    WAGTAILIMAGES_RENDITION_STORAGE = storages.backends.s3boto3.S3Boto3Storage
+
+This would store renditions in S3.
```

Comment:
can this line have a bit more detail, assume someone will copy this example above verbatim and have no idea what S3 is, maybe something like.

This assumes you are using the custom file [storage backend for Amazon S3](https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html).